src/error/metadata/codes/E042.md

```diff
@@ -0,0 +1,3 @@
+This error occurs when a schema file has lint rule violations
+
+The schema you're linting has violated some fo the rules configured for your graph. Fix the errors and rerun the lint command to verify the violations have been addressed.
```

Comment:
```suggestion
The schema you're linting has violated some of the rules configured for your graph. Fix the errors and re-run the lint command to verify the violations have been addressed.
```

Reply:
would be good to link to the lint documentation as well

Reply:
Updated text and added link